### PR TITLE
fix(command-status): fix typo Github -> GitHub

### PR DESCRIPTION
**Pull request by erezrokah**
**- Summary**

Follow up on https://github.com/netlify/cli/pull/3022.
Made another fix to the command and added a test

**- Test plan**

Added a new test

**- A picture of a cute animal (not mandatory but encouraged)**
🐈

### CODE REVIEW

File: src/commands/status/index.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class StatusCommand extends Command {
     const accountData = {
       Name: user.full_name,
       Email: user.email,
-      Github: ghuser,
+      GitHub: ghuser,
```
Comment (eduardoboucas) [inline src/commands/status/index.js:47]: 🙏🏻

**Review by eduardoboucas — APPROVED** [review]

**Comment by eduardoboucas** [comment]
The snapshot test is failing. Seems to be an issue with whitespace?

**Comment by erezrokah** [comment]
> The snapshot test is failing. Seems to be an issue with whitespace?

Yes! Looking into it ❤️

**Comment by erezrokah** [comment]
@eduardoboucas I [fixed](https://github.com/netlify/cli/pull/3024/commits/bffeca5166dfa3fad2febcaec326b8c5b42fb9d3) the test by using the `--json` flag. However, while doing so I realized this might be a breaking change if someone is accessing the `Github` property.

I'll see if we can get some data on how often its used.

**Comment by erezrokah** [comment]
> @eduardoboucas I [fixed](https://github.com/netlify/cli/pull/3024/commits/bffeca5166dfa3fad2febcaec326b8c5b42fb9d3) the test by using the `--json` flag. However, while doing so I realized this might be a breaking change if someone is accessing the `Github` property.
> 
> I'll see if we can get some data on how often its used.

Update: the `--json` flag is undocumented https://cli.netlify.com/commands/status/ so I think we're safe

**Pull request merged**